src/pages/Wardrobe/Wardrobe.tsx

```diff
@@ -41,22 +41,32 @@ function Wardrobe() {
 	]
 	type CategorySelection = 'Area' | 'Peep' | 'Outfit'
 
-	const { metadata } = useContext(MetadataContext)
+	const { metadata, setMetadata } = useContext(MetadataContext)
 	const [currentTraits, setCurrentTraits] = useState(sunIconTraits)
 	const [selectionString, setSelectionString] = useState('Area   Selections')
 	const [peepImage, setPeepImage] = useState('')
 	const [isBackgroundDisplayed, setIsBackgroundDisplayed] = useState(true)
 	const [isToggleHidden, setIsToggleHidden] = useState(true)
 	const [isShowingDone, setIsShowingDone] = useState(false)
 
-	if (!metadata) {
+	if (!metadata || !setMetadata) {
 		return <></>
 	}
 
 	useEffect(() => {
 		getPeepImage()
 	}, [metadata, isBackgroundDisplayed])
 
+	useEffect(() => {
+		const updatedMetadata: Trait[] = JSON.parse(JSON.stringify(metadata))
+		updatedMetadata.forEach(trait => {
+			if (trait.trait_type === 'Outerwear') {
+				trait.value = 'None'
+			}
+		})
+		setMetadata(updatedMetadata)
+	}, [])
+
```

Comment:
Before the `getPeepImage` `useEffect`?

Reply:
If I put it before then getPeepImage() doesn't get called when the metadata changes. So your first trait change will remove the hoodie and change whatever you chose.